api/v1alpha1/openapi.yaml

```diff
@@ -1801,6 +1801,14 @@ components:
           description: 'Current state of systemd units on the device.'
           items:
             $ref: '#/components/schemas/DeviceSystemdUnitStatus'
+        fingerprint:
```

Comment:
No need to add an extra field; the device's fingerprint == `device.name`.

Reply:
That makes sense but I was going off the `Data Definition Dictionary`[1] which describes it as a separate part of the status.

![image](https://github.com/flightctl/flightctl/assets/1249749/e70be2d2-fbfc-4556-a4a7-c60f761859fe)


[1] https://docs.google.com/spreadsheets/d/1xlslENyYVWL-M2Qw4lOwHOEWMXXk9cPN-tCb-IH7t3c/edit#gid=21336438